scripts/jenkins/Makefile.jenkins

```diff
@@ -485,3 +485,11 @@ check-leaks:
 		echo "Credentials leaked! Please check $(PWD)/leak-check.out"; \
 		exit 1; \
 	fi
+
+h2o-docker-build:
+    docker build -t h2oai/h2o-open-source:$(PROJECT_VERSION) -f docker/public/Dockerfile-h2o-release .
```

Comment:
`PROJECT_VERSION` is not available in nightly - we need to export that in nightly pipeline.